src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1245,5 +1245,40 @@ private void TestNormalize(SyntaxTriviaList trivia, string expected)
             var actual = trivia.NormalizeWhitespace("    ").ToFullString().NormalizeLineEndings();
             Assert.Equal(expected.NormalizeLineEndings(), actual);
         }
+
+        [Fact]
+        [WorkItem(60884, "https://github.com/dotnet/roslyn/issues/60884")]
+        public void TestNormalizeXmlArgumentsInDocComment1()
+        {
+            const string Expected = @"/// Prefix <b a=""x"" b=""y"">S_OK</b> suffix";
+            const string Text = @"/// Prefix <b    a=""x""  b=""y"" >S_OK</b> suffix";
+            TestNormalizeDeclaration(Text, Expected);
+        }
+
+        [Fact]
+        [WorkItem(60884, "https://github.com/dotnet/roslyn/issues/60884")]
+        public void TestNormalizeXmlArgumentsInDocComment2()
+        {
+            const string Expected = @"/// Prefix <b a=""x"" b=""y"">S_OK</b> suffix";
+            TestNormalizeDeclaration(Expected, Expected);
+        }
+
+        [Fact]
+        [WorkItem(60884, "https://github.com/dotnet/roslyn/issues/60884")]
+        public void TestNormalizeXmlArgumentsInDocComment3()
```

Comment:
Can you add a version of this that does not have a leading space before the `/>` in the original text?

Reply:
➡️ I have requested the other form in https://github.com/dotnet/roslyn/pull/61012#discussion_r866469118.